pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1211,6 +1211,23 @@ let
         };
       };
 
+      gencay.vscode-chatgpt = buildVscodeMarketplaceExtension {
+        meta = with lib; {
+          changelog = "https://marketplace.visualstudio.com/items/gencay.vscode-chatgpt/changelog";
+          description = "ChatGPT, GPT-3 and Codex conversations in Visual Studio Code";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=gencay.vscode-chatgpt";
+          homepage = "https://github.com/gencay/vscode-chatgpt";
+          license = licenses.isc;
```

Comment:
```suggestion
          license = lib.licenses.isc;
```

Reply:
There is a `with lib;` in the beginning.